services/copyright-service/.env

```
@@ -0,0 +1,51 @@
+NOTIFICATION_SERVICE_URL=http://localhost:8083
+
+# Security
+JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
+JWT_EXPIRES_IN=24h
+CORS_ORIGINS=http://localhost:3000,http://localhost:8080,http://localhost:3333
+RATE_LIMIT_WINDOW_MS=900000
+RATE_LIMIT_MAX=100
+
+# Web3
+WEB3_PROVIDER_URL=http://localhost:7545
+ACCOUNT_PRIVATE_KEY=0x855503bbff30f22be58aa5a251d8beea293c56abe91b9ebf7733158790739a6c
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)